### PR TITLE
Facebook Login Clone added

### PR DESCRIPTION
**Pull request by zalabhavy**
Closes: #352

- **Title** : : Facebook Login Clone
- **Name:** : Bhavy Zala
- **Idenitfy yourself:** : Gssoc'24

### Describe the add-ons or changes you've made 📃
I have added new folder Facebook_Login-Clone
Update data.json 
Update readme.md in frontend
Added readme.md in Facebook_Login-Clone file.

### How Has This Been Tested? ⚙️
Self checked

### Screenshots 📷
image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by zalabhavy** [comment]
@Avdhesh-Varshney Please check it and merge !

File: Front-end-Projects/README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@
 |   9   | [Apple Clone](./Intermediate/Apple-Clone )| ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
 |   10  | [Reddit Clone](./Intermediate/Reddit-Clone)| ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
   11  | [Microsoft Clone](./Intermediate/Microsoft-Clone)| ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
+  12 | [Facebook Login Clone](./Basic/Facebook_Login-Clone)| ![Basic]
+(https://img.shields.io/badge/Basic-00FF00?style=for-the-badge) |
```
Comment (Avdhesh-Varshney) [inline Front-end-Projects/README.md:40]: rectify it and keep it in 1 line. It distorted the table structure.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@zalabhavy folder name is to be `Facebook-Login-Page`. Update it.

**Comment by GAVINESHWAR** [comment]
@zalabhavy  I am  getting error while using nom start

PS C:\Users\Avineshwar G\Documents\internship\opensource\WebMasterLog> npm start
npm ERR! Missing script: "start"
npm ERR!
npm ERR! Did you mean one of these?
npm ERR! npm star # Mark your favorite packages
npm ERR! npm stars # View packages marked as favorites
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR! npm run

npm ERR! A complete log of this run can be found in: C:\Users\Avineshwar G\AppData\Local\npm-cache_logs\2024-06-21T05_25_24_324Z-debug-0.log

Can you say how you open the project

**Comment by Avdhesh-Varshney** [comment]
@GAVINESHWAR who are you? and what are you telling?

**Pull request closed without merging**